.gitignore

```diff
@@ -79,3 +79,6 @@ __pycache__
 # docker-compose artifacts
 /docker/buckets
 docker-compose.override.yml
+
+dist/
+.DS_Store
```

Comment:
usually advised to put those into your global gitignore, and only keep what actually belongs to the project in the repo's .gitignore, but this is just an opionion :)